databases.md

```diff
@@ -80,36 +80,28 @@ Connecting sets `mito:*connection*` to the new connection and returns it.
 
 Disconnect with `disconnect-toplevel`.
 
-=> you might make good use of a wrapper function:
+You might make good use of a wrapper function:
 
 ~~~lisp
 (defun connect ()
   "Connect to the DB."
-  (connect-toplevel :sqlite3 :database-name "myapp"))
+  (mito:connect-toplevel :sqlite3 :database-name "myapp"))
 ~~~
 
 ### Models
 
 #### Defining models
 
-In Mito, you can define a class which corresponds to a database table
-by specifying `(:metaclass mito:dao-table-class)`:
+In Mito, you can define a class which corresponds to a database table with the `deftable` macro:
 
 ~~~lisp
-(defclass user ()
-  ((name :col-type (:varchar 64)
-         :initarg :name
-         :accessor user-name)
-   (email :col-type (or (:varchar 128) :null)
-          :initarg :email
-          :accessor user-email))
-  (:metaclass mito:dao-table-class))
+(mito:deftable user ()
+  ((name :col-type (:varchar 64))
+   (email :col-type (or (:varchar 128) :null)))
 ~~~
+Alternatively, you can specify `(:metaclass mito:dao-table-class)` in a regular class definition.
 
-Note that the class automatically adds some slots: a primary key named `id`
-if there's no primary keys, `created_at` and `updated_at` for
-recording timestamps. To disable these behaviors, specify `:auto-pk
-nil` or `:record-timestamps nil` to the defclass forms.
+The `deftable` macro automatically adds some slots: a primary key named `id` if there's no primary key, and `created_at` and `updated_at` for recording timestamps. Specifying `(:auto-pk nil)` and `(:record-timestamps nil)` in the `deftable` form will disable these behaviours. A `deftable` class will also come with initializers and accessors for each named slot.
```

Comment:
Maybe say how the accessors are named and give an example? Is it `email` or `user-email`?